v3.0/reference/tools/sync-diff-inspector/overview.md

```diff
@@ -0,0 +1,214 @@
+---
+title: sync-diff-inspector 用户文档
+category: tools
```

Comment:
```suggestion
category: tools
aliases: ['/reference/tools/sync-diff-inspector']
```

Reply:
@dcalvin For this `aliases`, it should be `['docs-cn/v3.0/reference/tools/sync-diff-inspector/']`.

Reply:
added